math/realtime_stats.cpp

```diff
@@ -35,7 +35,7 @@ class stats_computer1 {
         n++;
         T tmp = x - K;
         Ex += tmp;
-        Ex2 += tmp * tmp;
+        Ex2 += int64_t(tmp) * tmp;
```

Comment:
This is an incorrect typecast. `tmp` is templated and hence can be either `float`, `int`, etc. 
Since `Ex2` is a `double` type, the correct cast would be to a `double`.
```suggestion
        Ex2 += static_cast<double>(tmp) * tmp;
```